docs/fidesops/docs/postman/Fidesops.postman_collection.json

```diff
@@ -4815,6 +4815,38 @@
 					},
 					"response": []
 				},
+				{
+					"name": "Authenticated Get Consent Preferences",
+					"request": {
+						"auth": {
+							"type": "bearer",
+							"bearer": [
+								{
+									"key": "token",
+									"value": "{{client_token}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n  \"phone_number\": \"{{phone_number}}\",\n  \"email\": \"{{email}}\"\n}"
+						},
+						"url": {
+							"raw": "{{host}}/consent-request/preferences",
+							"host": [
+								"{{host}}"
+							],
+							"path": [
+								"consent-request",
+								"preferences"
```

Comment:
URL and path don't match the URL on your endpoint, but I suspect that it is the postman collection that is right and the endpoint that is wrong.